### PR TITLE
Added isoly export, updated tests to use it.

### PR DESCRIPTION
**Pull request by simonmika**

### CODE REVIEW

File: isoly.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { CallingCode } from "./CallingCode"
```
Comment (tvartom) [inline isoly.ts:1]: I think all of these ca be changed to `export` directly. (And removing export-block.)

File: DateTime.spec.ts [hunk 2]
```diff
@@ -106,14 +106,14 @@ describe("DateTime", () => {
 				expect(isoly.DateTime.localize(new Date(date[0])).split(" ")[1]).toEqual(date[1])
 			)
 		it("localize DateTime with locale", () => {
-			expect(isoly.DateTime.localize("2020-12-31T23:59:59.000Z", "en-US")).toEqual("01/01/2021, 12:59:59 AM")
+			expect(isoly.DateTime.localize("2020-12-31T23:59:59.000Z", "en-US")).toEqual("01/01/2021, 12:59:59 AM")
```
Comment (tvartom) [inline DateTime.spec.ts:109]: So thosee are changed from `Space` to `Narrow No-Break Space`.
GitHub-tests seems to be fine (!?), however I get an error on my local machine running the tests.

Comment (tvartom) [reply DateTime.spec.ts:109]: Does your local machine expect Narrow No-Break Space?

File: index.ts [hunk 3]
```diff
@@ -1,3 +1,4 @@
+export * as isoly from "./isoly"
 import { CallingCode } from "./CallingCode"
```
Comment (tvartom) [inline index.ts:2]: I guess the old ones are preservred for backward compability.
Put a comment about the double ways to import, and which one that is prefered.

Comment (simonmika) [reply index.ts:2]: That pattern starts to be the normal practice.

**Review by tvartom — APPROVED** [review]

**Pull request merged**